README.md

```diff
@@ -6,6 +6,14 @@
 
 > Board support crate for HiFive1 and LoFive boards
 
+## Supported Boards
+
+`hifive1` - use feature `board-hifive1`
+`hifive1-revb` - use feature `board-hifive1-revb`
+`redv` - use feature `board-redv`
+`lofive` - use feature `board-lofive`
+`lofive-r1` - use feature `board-lofive-r1`
```

Comment:
Maybe it's a good idea to show their original names, maybe even with links

Reply:
Good point, I'll fish the links and push a commit